controllers/api/index.js

```diff
@@ -1,5 +1,6 @@
-const router = require('express').Router();
+const express = require('express');
+const app = express();
+const apiRoutes = require('./routes/apiRoutes')
```

Comment:
Line 3 is looking to import from this location
. = look in this folder
/routes = in folder called routes
/apiRoutes = look for a file called apiRoutes.js

There is no api/routes folder and no apiRoutes.js file.
To create a reference to the js file you did create would be `const jobSearchRoutes = require('./jobsearchroutes.js');`
Then add the line `router.use('/job-search', jobSearchRoutes);`

(Optional) Rename jobsearchroutes.js to either jobSearchRoutes.js(camelCase) or job-search-routes.js(kabob-case) in order to stay with the established folder naming convention.